### PR TITLE
Jfacorro.24.function.length

### PR DESCRIPTION
**Pull request by JakeQiu3**

### CODE REVIEW

File: src/elvis_style.erl [hunk 1]
```diff
@@ -833,6 +853,21 @@ is_ignored_var(Zipper) ->
         _OtherType -> false
     end.
 
+-spec check_exec_path_length(elvis_code:tree_node(), integer()) ->
+    [elvis_result:item_result()].
+check_exec_path_length(Node, MaxLength) ->
```
Comment (gadgetci) [inline src/elvis_style.erl]: According to **Xref**:
> `elvis_style:check_exec_path_length/2` calls undefined function `elvis_code:attr/2`

File: src/elvis_style.erl [hunk 2]
```diff
@@ -377,6 +380,23 @@ state_record_and_type(Config, Target, _RuleConfig) ->
             []
     end.
 
+-spec exec_path_length(elvis_config:config(),
+                               elvis_utils:file(),
+                               [list()]) ->
+    [elvis_result:item()].
+exec_path_length(_Config, Target, RuleConfig) ->
```
Comment (gadgetci) [inline src/elvis_style.erl]: According to **Xref**:
> `elvis_style:exec_path_length/3` calls undefined function `elvis_file:parse_tree/1`

File: src/elvis_style.erl (hunk 1 above)
Comment (gadgetci) [inline src/elvis_style.erl]: According to **Xref**:
> `elvis_style:check_exec_path_length/2` calls undefined function `ktn_code:longest_path/1`

Comment (elbrujohalcon) [inline src/elvis_style.erl]: For this work, https://github.com/inaka/elvis/blob/a79dc5f505d0678cddb06fd8b37d5f160c7a0009/src/elvis_code.erl#L185-L274 must be migrated to `ktn_code`

**Comment by jfacorro** [comment]
@JakeQiu3 The code in this branch is not finished, see the discussion [here](https://github.com/inaka/elvis/issues/24#issuecomment-50157915). If you would like to take on the challenge we can chat on Inaka's [open source room](http://inaka.net/hipchat).

**Pull request closed without merging**